### ISSUE TITLE
How to add a list to a page with scrolling?

### ISSUE THREAD

**Issue by cestrensem** [bvaughn/react-window#193]
Hello, i have a page with large amount of comments.
![image](https://user-images.githubusercontent.com/3720256/55288397-692a2100-53b7-11e9-8c2b-bcb6ea6dbe89.png)

But this page have it's own scrolling, so I can't add comment thread with fixed width. 

Which way should i go then?

**Comment by bvaughn** [comment]
I'm not sure this sort of question should be asked in a GitHub issue. I think Stack Overflow might be more appropriate, but...

Vertical lists doesn't require a fixed width. You can use a percentage width for them. They do require a fixed height though.

If you meant to type "fixed height", then I'd say this sounds like a duplicate of #192 and I'll re-post my answer from that issue:

---

What your'e describing isn't supported by react-window. It was supported by react-virtualized, via a `WindowScroller` component, but `WindowScroller` was one of those painful to maintain components. I see the usefulness of it, but I'm not sure I'm interested in committing to maintain another one. I would prefer instead, if possible, for an external component to be built (and published separately) that implements this functionality on top of react-window.

I might be willing to make a couple of changes to react-window to support such a component, if it existed, depending on how complex they would be.

---

If you're willing to put together a proposal (demo code, proposed API changes) I'm willing to talk it over and consider potential, minor changes to this library– but I'm confident that I'm not interested in the following at this time:
* Adding a `WindowScroller` (or similar) component to the react-window package. I don't have the time or desire to take on the maintenance burden of such a component.
* Adding "controlled" props for `scrollLeft` and `scrollTop`. That sort of an API (mix of controlled an uncontrolled props for scroll offset) [was a big regret of mine with react-virtualized](https://twitter.com/brian_d_vaughn/status/959600888242307072?lang=en. It's best to either go fully controlled (no state, only props) or fully uncontrolled (only state, with maybe a `props.defaultValues` that's read only once before mounting). react-window already has an uncontrolled API for this (with imperative methods for setting an overridden value).

I'm going to close this issue now because I don't plan on taking any direct action, but we can continue the discussion here 😄
